pydicom/dataset.py

```diff
@@ -1754,6 +1776,351 @@ def walk(self, callback, recursive=True):
                     for dataset in sequence:
                         dataset.walk(callback)
 
+    @classmethod
+    def _convert_to_python_number(cls, value, vr):
+        """Makes sure that values are either ints or floats
```

Comment:
I'm wondering if this needs to be a class method.  It seems to use cls only to access the _VRs_TO_BE_INTS, etc, which don't seem to be used elsewhere.  Could some functions like this be moved into a separate json.py module as a module method, for example? (Dataset code is already very large, so it would be nice to keep it to things that really require access to the class or instance variables).